src/quo2/components/settings/data_item/style.cljs

```diff
@@ -0,0 +1,68 @@
+(ns quo2.components.settings.data-item.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container
+  [size card? blur?]
+  {:flex               1
+   :flex-direction     :row
+   :justify-content    :space-between
+   :padding-vertical   (when (= size :default) 8)
+   :padding-horizontal (when (= size :default) 12)
+   :border-radius      16
+   :border-width       (when (and card? (not= size :small)) 1)
+   :border-color       (if
+                         blur?
+                         colors/white-opa-10
+                         (colors/theme-colors colors/neutral-10
+                                              colors/neutral-80))})
+
+(defn loading-container
+  [size blur?]
+  {:width            (if (= size :default) 132 72)
+   :height           (if (= size :default) 16 10)
+   :background-color (if
+                       blur?
+                       colors/white-opa-5
+                       (colors/theme-colors colors/neutral-5
```

Comment:
same here, can you check all uses of `theme-colors` in this pr 👍

Reply:
Done